examples/notebooks/keras_private_predict.ipynb

```diff
@@ -0,0 +1,365 @@
+{
```

Comment:
<p>is this notebook still useful here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tf-encrypted/tf-encrypted/pull/571/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Good point, it should be removed.